javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,103 @@
+//Item array removal
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Rasmus";
+names.splice(names.indexOf(nameToRemove), 1);
+console.log(names); // [ "Peter", "Ahmad", "Yana", "kristina", "Samuel", "katrine", "Tala" ]
+
+//When will we be there??
+
+const travelInformation = {
+  speed: +prompt("enter your speed in km/h"),
+  destinationDistance: +prompt("enter your distants in km"), // a small change
+};
+function calculateTime(distant, speed) {
+  const travelTime = distant / speed;
+  const hours = Math.floor(travelTime);
+  const minutes = ((travelTime - hours) * 60).toFixed(0);
+  return `${hours} hours and ${minutes} minutes`;
+}
+console.log(
+  calculateTime(travelInformation.speed, travelInformation.destinationDistance)
+); // 8 hours and 38 minutes
+
```

Comment:
It's nice that you used a prompt here, so users can use different inputs and see different results, however, this code is not logging the expected result: `8 hours and 38 minutes`. This is happening because you are using the `distant` and `speed` as parameters instead of `travelInformation`. To fix this, use the `travelInformation`as a parameter, and access the object properties of this variable by using the `.` inside the `travelTime`. 
Ex.: `function calculateTime(travelInformation) {
  const travelTime = travelInformation.destinationDistance / travelInformation.speed;`. In the `console.log`, log only the `travelInformation`. Ex.: `console.log(travelInformation)`. Doing that, you will get the hours and minutes as expected. 😊